tests/frontend/test_users.py

```diff
@@ -229,6 +232,64 @@ def test_user_regular_notifications(base_url, selenium, variables):
         count += 1
 
 
+@pytest.mark.serial
+@pytest.mark.nondestructive
+def test_user_update_profile(base_url, selenium, variables):
+    user = User(selenium, base_url).open().wait_for_page_to_load()
+    user.login('reusable_user')
+    updated_name = 'new_display_name'
+    # update field
+    user.edit.display_name_field.clear()
+    user.edit.display_name_field.send_keys(updated_name)
+    # clear field
+    user.edit.location_field.clear()
+    # append to field text
+    user.edit.biography(variables['biography_extra'])
+
+    user.edit.submit_changes()
+    user.wait.until(
+        EC.visibility_of_element_located((By.CLASS_NAME, 'UserProfile-name'))
+    )
```

Comment:
we have the method `wait_for_user_to_load` in `users,py` that does what your wait here is supposed to do.